src/server.prod.js

```diff
@@ -59,6 +59,7 @@ class ProdServer {
 
     app.use('/healthcheck', healthcheck());
     app.use(...getMiddleWares({ renderFn, before, after, nock, nockPath, accessLogs }));
+    app.set('trust proxy', this.config.trustProxy);
```

Comment:
where is this variable accessed ?

Reply:
It's a setting for the Express app, see https://expressjs.com/de/api.html#app.settings.table